unime/src/lib/events/HistoryEntry.svelte

```diff
@@ -1,34 +1,75 @@
-  import type { DisplayCredential } from '@bindings/display-credential/DisplayCredential';
+  import { goto } from '$app/navigation';
+  import { formatDistanceToNow, type Locale } from 'date-fns';
+  import { de, enGB, enUS, nl } from 'date-fns/locale';
+
+  import type { HistoryCredential } from '@bindings/HistoryCredential';
 
   import ListItemCard from '$lib/components/molecules/ListItemCard.svelte';
-  import type { Connection } from '$lib/connections/types';
   import { state } from '$src/stores';
 
-  // export let connection: Connection;
   export let title: string;
-  export let timestamp: string;
-  export let credentials: DisplayCredential[] = [];
+  export let date: string;
+  export let credentials: HistoryCredential[] = [];
+
+  function getRelativeDate() {
+    let locale: Locale;
+
+    switch ($state.profile_settings.locale) {
+      case 'en-US': {
+        locale = enUS;
+        break;
+      }
+      case 'en-GB': {
+        locale = enGB;
+        break;
+      }
+      case 'de-DE': {
+        locale = de;
+        break;
+      }
+      case 'nl-NL': {
+        locale = nl;
+        break;
+      }
+    }
+
+    function capitalizeFirstLetter(str: string) {
+      return str.charAt(0).toUpperCase() + str.slice(1);
+    }
+
+    const dateStr = formatDistanceToNow(new Date(date), { locale, addSuffix: true });
+    return capitalizeFirstLetter(dateStr);
+  }
+
+  function navigateToCredential(credentialId: string) {
+    const credential = $state.credentials.find((cred) => cred.id === credentialId);
+
+    if (credential) {
+      if (credential.data.type.includes('OpenBadgeCredential')) {
+        goto(`/badges/${credential.id}`);
+      } else {
+        goto(`/credentials/${credential.id}`);
+      }
+    }
+  }
 </script>
 
 <div class="flex flex-col">
-  <p class="text-[14px]/[22px] font-medium text-slate-800">{title}</p>
-  <p class="text-[12px]/[20px] font-medium text-slate-500">
-    {new Date(timestamp).toLocaleString($state.profile_settings.locale, {
-      dateStyle: 'medium',
-      timeStyle: 'medium',
-    })}
+  <p class="text-[14px]/[22px] font-medium text-slate-800 dark:text-grey">{title}</p>
+  <p class="text-[12px]/[20px] font-medium text-slate-500 dark:text-slate-300">
+    {getRelativeDate()}
   </p>
   {#if credentials.length > 0}
-    <div class="mt-[12px] rounded-xl border border-slate-200 bg-white p-[3px] dark:border-slate-600">
+    <div class="mt-[12px] rounded-xl border border-y-gray-200 bg-white p-[3px] dark:border-dark dark:bg-dark">
       {#each credentials as credential}
         <ListItemCard
           id={credential.id}
-          title={credential.metadata.display.name ?? credential.data.type.at(-1)}
-          description="Lorem ipsum dolor"
-        >
-          <!-- <span slot="icon"><EnvelopeSimple class="h-6 w-6" /></span> -->
-        </ListItemCard>
+          title={credential.title}
+          description={credential.sub_title}
+          class="rounded-none"
```

Comment:
Why remove the rounded borders?
* It does not have a visual effect since background and element have the same color.
* Also when they don't have the same color, it looks worse without rounded borders.